Library/Homebrew/cmd/update.rb

```diff
@@ -57,6 +58,7 @@ def update
     else
       puts "Updated Homebrew from #{master_updater.initial_revision[0,8]} to #{master_updater.current_revision[0,8]}."
       report.dump
+      Descriptions.refresh_cache
```

Comment:
I thought the only large impact would be the initial creation of the cache. Would normal updates really be that huge?

If yes, perhaps drop this from `update` and only refresh the cache when the user calls `brew desc` explicitly? That way, users who want this will wait, but nobody else will?